src/js/providers/flash.js

```diff
@@ -338,6 +338,10 @@ define([
                         this.trigger(events.JWPLAYER_MEDIA_ERROR, event);
                     }, this);
 
+                    _swf.on('adaptiveEventAudio', function() {
+                        this.trigger('mediaType', {mediaType: 'audio'});
+                    }, this);
```

Comment:
`this.sendMediaType(item.sources);` is called in both providers' `load` function. This occurs before the `adaptiveEventAudio` event and always sends `{mediaType: 'video'}` for HLS streams.